swagger/src/main/java/grpcbridge/swagger/PathBuilder.java

```diff
@@ -0,0 +1,37 @@
+package grpcbridge.swagger;
+
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.Descriptors.FieldDescriptor;
+
+import java.util.List;
+
+public class PathBuilder {
+    private final String path;
+    private final List<FieldDescriptor> pathParameters;
+    private final SwaggerConfig config;
+
+    private PathBuilder(
+            String path,
+            List<FieldDescriptor> pathParameters, SwaggerConfig config) {
+        this.path = path;
+        this.pathParameters = pathParameters;
+        this.config = config;
+    }
+
+    public static PathBuilder create(
+            String path,
+            List<FieldDescriptor> pathParameters,
+            SwaggerConfig config) {
+        return new PathBuilder(path, pathParameters, config);
+    }
+
+    public String getPath() {
+        String newPath = path;
+        for (Descriptors.FieldDescriptor fieldDescriptor : pathParameters) {
+            newPath = newPath.replace(
+                    String.format("{%s}", fieldDescriptor.getName()),
+                    String.format("{%s}", config.formatFieldName(fieldDescriptor)));
```

Comment:
Not sure I follow. Why do we need to delay using the formatter for path parameters until now instead of formatting them when we insert them?

Reply:
well, I tried to avoid changing BridgeRule. This is only used for swagger gen only. And on other side, parameters builder would not be wrong place for this ?

Reply:
I can move logic to `parametersBuilder` with making  `BridgeHttpRule.path` settable, but again I don't think it is supposed to be there.